packages/jellyfish-api-core/src/category/spv.ts

```diff
@@ -78,3 +94,21 @@ export interface SendMessageResult {
   txid: string
   sendmessage: string
 }
+
+export interface CreateHtlcOptions {
+  /** Timeout of the contract (denominated in blocks) relative to its placement in the blockchain. Minimum 9. See HTLC_MINIMUM_BLOCK_COUNT */
+  timeout: string
+  /** SHA256 hash of the seed. If none provided one will be generated */
+  seed?: string
+}
+
+export interface CreateHtlcResult {
+  /** The value of the new Bitcoin address */
+  address: string
+  /** Hex-encoded redemption script */
+  redeemScript: string
+  /** Hex-encoded seed */
+  seed: string
```

Comment:
do this https://github.com/DeFiCh/jellyfish/pull/511/files#r680662678 and check whether `seed` is optional. and update accordingly.

Reply:
good catch ! Updated the interface and set seed and seedhash as optionals